packages/nimble-components/src/table-column/enum-base/models/mapping-key-resolver.ts

```diff
@@ -0,0 +1,30 @@
+import { nullableNumberConverter } from '@microsoft/fast-element';
+import type { MappingKey } from '../../../mapping/base/types';
+import type { MappingKeyType } from '../types';
+
+/**
+ * Converts a Mapping key (which is a string when configured in HTML) to the
+ * given keyType. The converted value can then be used to compare against
+ * values in the table data.
+ */
+export const resolveKeyWithType = (
+    key: MappingKey | undefined,
+    keyType: MappingKeyType
+): MappingKey | undefined => {
+    if (keyType === 'number') {
+        const converted = nullableNumberConverter.fromView(key) as
+            | number
```

Comment:
Is that first `|` (in front of `number`) necessary?

Reply:
Yes, the formatter adds it back if I remove it.